components/fullscreen-image-quote/Component.jsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable react/jsx-no-target-blank */
+import React from "react";
+import { Container } from "../../packages/grids/Container";
+import { FAIcon } from "../../packages/icons/FAIcon";
+import * as styles from "./styles";
+
+/**
+ * Fullscreen Image Quote component
+ *
+ * @param {string} quote
+ * The quote/content text
+ *
+ * @param {string} quoteHAlign
+ * The horizontal alignment of the quote: "left" | "right"
+ *
+ * @param {string} quoteVAlign
+ * The vertical alignment of the quote: "top" | "center" | "bottom"
+ *
+ * @param {boolean} removeTopSpacing
+ * When this is true, the component will be shifted up to eliminate the spacing between itself and the component above.
+ * This is useful when several Fullscreen Image Quotes are stacked on top of each other.
+ *
+ * @param {object} imageData
+ * The data of the background image
+ *
+ * @param {string} imageVPosition
+ * The crop position (object-position) of the image: "top" | "center" | "bottom"
+ *
+ * @param {object} mobileImageData
+ * The data of the background image for mobile
+ *
+ * @param {string} internalLinkUrl
+ * The internal link URL
+ *
+ * @param {object} ctaDetails
+ * The different fields for the CTA
+ *
+ * @returns {JSX.Element}
+ * @constructor
+ */
+
+export default function FullscreenImageQuote({
+  quote,
+  quoteHAlign,
+  quoteVAlign,
+  removeTopSpacing,
+  imageData,
+  imageVPosition,
+  mobileImageData,
+  internalLinkUrl,
+  ctaDetails,
+}) {
+  const { ctaPreText, ctaText, ctaSubtext, externalUrl, isNewWindow } =
+    ctaDetails;
+  // Check externalUrl field to see if it is actually external
+  // Do not use external arrow icon and rel attributes if link has news.stanford.edu
+  const isRealExternalLink =
+    !!externalUrl && !externalUrl?.includes("news.stanford.edu");
```

Comment:
The CTA uses our usual external link check as usual. It uses the external link icon and animation when the link is external. I figured we would want the UX to be consistent on the site.